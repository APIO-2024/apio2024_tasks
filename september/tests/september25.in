txy4h26c1rm1uv8tr3eonahd67u8h56x
53
10 1
0 0 2 1 3 5 5 4 1
6 7 5 8 2 3 9 1 4
10 1
0 1 0 3 0 2 6 5 5
9 7 8 6 2 5 4 1 3
10 1
0 0 1 2 2 1 1 6 3
8 9 5 7 6 3 4 2 1
10 1
0 0 1 3 4 2 0 5 5
8 7 6 2 5 4 3 9 1
10 1
0 0 1 3 2 1 0 5 7
9 7 8 6 3 4 1 5 2
10 1
0 0 0 0 2 5 0 0 7
6 3 9 8 1 5 4 2 7
10 1
0 0 0 0 2 0 6 1 1
7 3 6 5 8 4 9 1 2
10 1
0 1 2 1 0 0 1 2 1
9 8 4 5 6 7 3 1 2
10 1
0 1 1 3 0 0 6 4 0
2 5 8 7 4 3 9 1 6
10 1
0 1 2 2 3 1 3 1 7
5 4 6 8 9 7 3 1 2
10 1
0 1 2 2 2 1 2 2 5
6 4 8 9 5 3 7 1 2
10 1
0 1 2 3 2 0 2 3 3
8 4 7 6 5 9 3 2 1
10 1
0 1 2 1 3 4 1 4 5
7 6 9 8 5 3 2 4 1
10 1
0 1 0 1 4 5 1 5 8
6 3 9 2 8 5 4 7 1
10 1
0 0 1 1 3 0 6 1 7
4 5 9 8 2 3 7 6 1
10 1
0 1 0 0 1 0 5 1 2
6 8 3 4 9 5 7 1 2
10 1
0 0 2 1 3 2 6 0 8
4 9 8 5 1 3 7 6 2
10 1
0 1 1 0 1 5 4 0 0
6 3 5 1 2 9 7 4 8
10 1
0 0 1 1 3 1 5 2 0
6 4 8 2 9 7 5 3 1
10 1
0 0 2 2 3 1 2 2 7
5 4 3 6 8 9 1 7 2
40 1
0 1 1 2 3 3 4 2 1 6 9 3 3 12 12 4 8 16 8 11 9 3 7 13 9 22 4 25 1 11 23 26 18 2 28 28 19 23 7
5 20 21 30 33 37 24 10 29 6 13 27 11 36 17 32 31 19 26 22 34 14 18 39 16 35 15 25 8 9 28 38 23 7 12 4 2 3 1
40 1
0 1 2 0 2 5 4 4 3 9 2 6 2 8 6 8 5 6 9 1 12 6 22 1 6 4 23 7 5 25 12 16 29 13 30 2 5 25 33
21 15 37 32 39 35 31 30 24 16 20 28 7 36 27 17 26 11 33 38 12 14 23 34 29 22 19 13 18 10 8 25 9 3 4 6 5 1 2
40 1
0 1 1 1 1 3 0 0 0 4 9 1 1 9 13 4 13 7 17 9 19 10 20 19 13 0 2 27 14 9 26 24 2 9 0 29 11 30 24
35 12 33 6 32 28 36 16 25 27 8 21 22 5 10 31 37 14 23 29 34 38 18 20 7 4 39 11 3 26 24 2 19 17 30 15 13 9 1
40 1
0 0 1 2 2 5 0 7 8 3 5 11 3 12 0 11 8 7 5 17 8 13 19 1 11 24 6 9 11 12 23 20 21 8 12 33 34 23 10
18 38 35 25 14 15 4 39 16 29 26 32 30 22 36 24 37 20 28 31 33 17 9 12 21 34 10 8 27 11 13 23 6 3 19 1 7 5 2
40 1
0 1 0 3 1 0 5 4 2 0 0 0 4 6 9 0 8 14 8 9 8 6 20 11 16 2 14 16 28 22 27 15 21 10 20 16 19 17 36
26 33 31 25 12 39 38 36 29 30 34 22 18 21 32 27 23 37 14 10 7 13 19 15 11 24 28 5 6 20 35 16 9 2 17 1 8 4 3
100 1
0 1 2 3 2 2 0 6 4 1 8 1 7 1 12 11 8 11 8 0 12 16 13 15 17 0 7 18 9 8 2 4 30 7 27 25 0 8 0 4 9 6 14 10 20 5 32 37 32 27 23 14 34 14 36 1 40 0 14 30 12 22 57 7 46 57 4 47 33 24 1 24 42 54 37 26 63 33 47 37 15 39 68 72 71 18 72 18 23 14 12 60 63 20 73 40 53 83 27
21 78 19 82 89 99 98 74 67 66 93 90 39 85 75 52 59 69 48 38 58 56 81 51 65 61 49 44 83 79 91 68 46 55 71 54 43 94 35 87 62 31 14 80 36 70 10 92 86 97 41 60 22 23 45 53 25 20 13 17 33 64 34 76 5 50 95 16 84 37 77 63 28 88 96 27 24 30 72 26 47 7 15 57 12 9 29 11 40 18 32 8 4 73 42 6 2 3 1
1000 1
0 0 2 0 1 0 4 1 8 8 10 4 9 5 4 8 13 3 11 6 0 11 9 2 16 14 15 11 5 23 29 24 18 21 20 18 22 33 9 37 33 37 12 29 17 36 28 15 23 30 48 48 10 19 5 7 14 49 6 53 16 8 15 35 21 36 24 3 66 59 46 23 5 40 14 62 60 64 54 54 20 12 11 51 57 33 79 6 38 45 70 14 70 60 14 2 17 53 24 36 34 26 11 34 30 78 63 93 87 105 16 12 11 91 69 24 10 95 52 96 34 6 100 3 88 1 12 14 12 120 20 68 130 96 47 36 38 15 3 41 93 34 75 108 30 49 141 50 82 19 95 72 138 88 18 50 2 74 81 63 74 103 0 10 133 9 3 66 20 115 100 44 53 47 13 17 55 171 176 35 174 159 30 112 170 98 86 187 36 173 158 136 145 188 83 17 17 6 143 177 122 162 15 36 49 93 149 37 160 49 155 95 57 149 109 52 14 88 196 148 205 46 132 53 94 15 32 212 51 214 68 223 69 182 182 117 83 142 82 88 152 62 79 80 212 65 56 181 20 154 35 21 104 153 227 138 108 39 155 41 251 188 77 16 13 113 69 112 141 52 211 19 182 265 202 17 202 23 55 225 172 273 217 151 227 271 193 239 280 16 134 287 123 267 4 22 52 154 136 98 211 86 29 28 116 287 297 64 277 91 270 177 113 43 249 144 53 228 268 147 289 219 108 319 262 178 34 93 301 91 33 266 313 129 128 96 310 262 249 276 174 111 33 190 320 22 102 337 287 87 35 244 297 124 169 24 156 352 160 191 293 187 165 310 220 169 247 94 145 335 94 147 293 311 52 128 243 186 374 273 77 119 231 44 286 324 339 134 78 332 162 326 356 158 5 139 298 69 225 154 57 26 32 230 361 368 311 347 34 399 381 8 310 194 387 353 399 92 149 110 263 108 206 267 207 183 298 281 392 249 161 323 197 252 89 247 330 297 414 296 430 55 189 6 9 432 250 181 375 150 133 205 283 425 440 216 131 112 322 80 327 256 254 169 140 380 439 9 122 418 162 442 8 375 472 337 340 227 36 403 297 172 236 61 232 378 149 71 479 126 127 85 199 463 251 491 454 402 146 164 236 19 228 10 293 486 468 178 491 365 139 301 163 170 54 312 411 163 90 398 215 398 117 490 150 21 92 355 221 313 96 323 192 284 17 300 66 495 31 284 521 442 216 52 16 68 264 120 220 176 266 96 194 316 333 552 393 94 270 38 21 267 298 511 377 547 202 157 313 314 280 329 32 442 413 547 6 46 437 346 2 550 528 422 289 532 45 347 521 296 254 31 431 461 592 460 496 155 270 136 444 198 85 70 501 6 345 247 446 169 233 23 344 509 10 414 122 368 67 37 592 62 568 413 437 579 312 306 574 311 369 162 501 409 628 321 41 324 5 329 162 331 512 100 139 329 570 642 19 207 403 8 652 367 6 546 70 504 332 233 185 230 255 488 376 56 571 658 248 451 55 190 91 336 333 26 243 567 304 316 76 611 380 486 544 181 397 300 536 411 241 579 105 278 549 81 163 508 89 541 50 689 245 22 297 192 264 301 368 115 542 605 439 235 506 70 18 280 211 448 412 303 454 239 698 287 43 219 499 314 413 356 100 96 157 136 404 515 571 314 466 311 475 650 398 340 396 603 51 678 739 675 338 241 340 468 389 637 606 641 338 95 679 499 343 687 308 333 313 420 63 16 181 339 498 406 372 724 767 190 775 63 723 218 552 216 131 713 61 190 240 421 188 269 229 138 197 38 733 226 478 5 680 23 613 12 641 565 527 628 35 772 509 264 354 785 755 366 790 314 507 759 413 47 269 346 117 542 547 186 471 563 307 109 78 764 251 80 373 413 622 361 175 746 41 99 639 488 340 15 318 243 151 838 34 43 142 634 714 83 712 830 551 520 583 450 421 358 401 416 652 297 281 744 182 790 758 703 838 33 76 774 178 694 279 873 854 72 556 305 538 847 735 601 579 475 148 752 180 48 553 382 388 416 220 410 418 122 63 896 566 61 712 523 40 140 476 318 908 41 845 454 648 492 140 782 448 902 400 305 132 66 67 194 620 472 344 899 871 713 285 93 538 519 625 916 530 97 890 60 680 15 82 325 387 881 562 496 43 463 651 467 88 374 77 640 736 654 862 777 73 215 904 23 103 592 131 786 375 873 579 977 265 529 739 89 244 130 48 185 403 522 28 695 48 542 14 622 390
837 288 274 441 482 572 980 668 489 646 893 945 328 543 635 292 258 415 827 505 617 991 986 179 816 537 783 943 953 125 796 621 533 952 686 760 443 982 106 168 363 883 907 208 778 848 875 238 539 395 728 721 670 923 718 209 681 766 604 121 424 167 452 585 213 707 950 785 906 427 977 956 887 455 805 966 688 852 693 940 600 667 983 234 949 941 829 386 919 394 614 813 996 745 500 748 445 732 291 933 702 905 771 474 731 795 569 655 591 477 222 960 761 915 910 882 854 924 577 593 42 898 921 874 857 430 692 564 405 700 224 885 581 756 803 890 999 548 629 383 930 856 927 787 808 603 526 309 976 27 676 773 922 985 357 447 503 815 305 804 509 851 237 517 711 858 410 936 575 351 958 166 833 657 798 573 609 896 315 973 596 282 850 845 867 469 705 391 834 826 137 687 743 872 704 749 989 801 789 516 853 457 831 752 535 242 846 475 691 307 792 397 142 819 781 165 246 295 260 865 131 647 257 334 863 706 979 480 598 189 428 299 555 900 231 390 666 868 472 797 421 811 993 449 451 360 727 464 277 576 912 589 99 935 433 317 487 962 638 107 594 423 116 631 717 988 580 371 861 799 723 944 250 272 584 582 968 701 769 914 412 65 747 715 971 818 750 481 774 358 814 938 775 768 261 969 659 997 39 645 285 864 200 901 302 191 649 129 471 73 259 877 965 362 839 880 557 519 984 697 253 502 610 957 513 435 840 616 656 534 741 375 807 730 735 497 626 886 788 740 684 908 884 763 210 90 465 510 779 820 682 422 342 824 891 370 346 800 111 776 975 438 461 632 514 454 612 559 881 583 335 528 350 565 434 673 587 959 369 406 911 558 560 928 524 719 716 417 373 653 204 203 809 494 104 492 623 994 849 961 823 722 618 327 679 633 963 878 459 675 920 229 793 520 349 366 836 456 678 498 869 931 333 408 164 135 156 511 290 770 860 399 742 541 620 762 648 76 265 939 393 611 683 917 643 615 918 426 453 855 396 784 352 651 470 608 992 553 954 619 630 677 998 870 446 460 821 599 550 720 843 546 485 522 964 429 574 713 118 873 184 365 634 990 554 473 590 951 734 419 479 101 725 463 709 642 765 889 84 556 525 377 364 226 708 51 324 402 436 281 58 275 780 158 613 552 897 523 218 283 418 967 904 206 934 252 570 254 806 899 172 698 736 467 978 903 879 812 385 987 680 355 946 379 895 841 764 508 660 758 567 690 791 607 466 561 937 195 518 729 925 661 926 384 368 859 325 955 699 169 484 825 139 862 392 540 838 401 866 817 374 696 802 512 180 871 737 830 902 835 664 488 636 462 767 344 316 672 407 671 183 588 303 578 341 744 842 440 233 202 733 641 347 25 695 439 909 308 144 545 527 133 627 420 654 710 331 359 506 387 888 163 398 45 981 624 102 312 726 61 714 339 531 622 755 507 751 243 249 197 232 450 586 256 942 929 286 388 669 703 245 416 782 380 995 326 876 974 579 663 314 640 916 694 499 486 114 71 85 483 177 110 185 311 685 724 356 83 597 44 296 62 414 602 294 201 332 279 271 496 652 175 972 894 150 832 361 122 491 568 794 319 665 284 376 772 194 413 913 240 490 318 293 404 544 322 192 754 403 790 157 134 400 198 219 529 932 130 947 178 432 538 532 354 367 448 378 674 151 244 566 46 738 124 221 712 948 176 268 273 689 409 263 323 170 223 521 844 270 136 255 34 171 190 278 217 382 132 186 115 146 92 43 173 196 248 493 739 247 345 329 892 269 215 828 515 239 753 411 321 786 228 212 320 601 970 746 571 127 57 536 542 109 822 551 658 56 105 300 97 67 530 89 126 637 336 123 205 381 128 289 340 442 98 95 160 644 276 650 348 100 7 21 759 757 605 847 199 606 777 562 639 431 162 501 662 563 338 80 266 140 625 495 77 372 147 120 153 55 458 353 87 108 216 119 504 343 298 181 547 161 63 96 154 304 478 437 17 141 138 236 68 79 174 117 38 389 227 230 264 301 103 41 54 425 74 211 313 32 113 16 40 267 24 330 143 251 93 280 262 214 810 37 13 207 444 75 188 476 112 235 241 22 152 628 26 19 187 297 225 52 14 49 149 69 468 337 94 66 78 64 82 86 182 155 47 28 595 159 306 287 310 549 11 12 81 60 220 193 48 35 15 91 33 70 148 53 145 20 4 36 10 72 592 88 31 59 18 3 2 6 29 50 5 23 30 9 8 1
100 1
0 1 0 1 2 2 2 4 3 7 1 6 4 6 14 13 4 13 2 2 10 12 9 10 2 6 12 8 18 2 24 26 29 30 8 21 34 26 12 4 18 21 27 6 21 18 18 3 7 46 0 32 35 30 15 50 7 55 54 3 51 18 13 2 46 16 20 6 68 19 17 66 43 8 52 54 14 2 25 44 79 17 0 70 20 52 22 43 50 74 39 84 88 49 82 15 36 65 61
95 96 45 58 81 82 86 28 64 47 87 60 78 85 72 73 53 22 48 71 99 79 98 17 59 80 42 63 31 35 93 56 91 62 25 5 77 23 69 88 97 76 61 54 89 68 33 66 37 67 43 11 94 44 51 50 75 57 16 55 52 15 9 27 65 21 36 34 14 46 90 49 3 20 39 74 12 92 84 70 32 83 38 24 10 40 26 6 19 30 8 7 29 2 41 18 4 13 1
40 1
0 0 2 1 2 2 1 0 0 4 1 7 0 5 3 4 11 4 2 16 0 20 22 12 17 23 26 25 18 15 21 7 23 5 24 10 3 37 2
34 36 39 6 9 35 19 38 37 29 31 13 30 10 8 15 33 27 3 24 28 26 12 22 23 14 5 18 32 21 20 25 16 17 7 2 11 4 1
40 1
0 0 1 1 2 3 6 0 4 7 10 5 0 6 13 15 6 4 3 10 7 1 17 0 21 16 7 16 5 29 12 20 23 25 33 5 32 35 4
22 31 11 30 29 27 9 39 19 28 24 38 36 26 35 34 37 25 12 33 14 20 32 16 5 2 8 23 21 17 18 15 10 13 7 6 4 3 1
40 1
0 0 1 2 1 3 6 5 3 1 9 0 10 6 3 1 11 4 1 11 11 8 9 17 10 20 10 18 26 3 29 26 10 26 28 12 4 4 27
35 25 14 19 30 23 16 38 32 22 8 37 5 36 13 15 21 39 31 29 33 27 12 10 34 28 24 26 17 7 18 6 4 20 11 9 2 3 1
40 1
0 0 2 0 2 4 0 5 1 4 0 0 9 7 1 12 15 7 5 8 15 7 0 12 24 25 16 27 26 11 26 1 6 11 20 32 0 19 6
37 30 36 31 21 28 14 35 32 38 23 3 17 22 39 10 13 15 20 9 34 27 19 11 18 33 7 1 29 6 26 4 8 16 5 25 2 24 12
40 1
0 0 2 2 4 2 6 0 1 5 0 8 4 7 2 9 15 17 9 0 0 2 21 6 23 25 4 9 0 9 5 10 13 19 17 1 26 30 0
16 38 35 27 14 31 7 3 36 29 24 22 33 18 13 20 6 30 28 37 12 32 39 17 8 15 10 11 26 25 23 34 5 4 19 9 1 21 2
10 1
0 0 0 3 0 5 2 6 3
9 4 7 2 1 8 3 5 6
10 1
0 1 2 1 0 0 1 0 7
9 3 4 5 8 2 6 7 1
10 1
0 1 1 3 1 4 3 5 5
8 7 2 6 4 9 5 3 1
10 1
0 1 2 1 2 3 2 0 0
9 8 4 5 7 3 2 6 1
10 1
0 0 0 2 0 4 1 5 3
8 5 9 6 2 4 3 7 1
10 1
0 1 1 0 0 1 6 7 8
3 5 9 2 4 8 7 6 1
10 1
0 1 2 0 2 4 5 7 4
3 8 6 9 5 4 7 1 2
10 1
0 0 0 2 4 0 0 5 1
3 8 5 6 1 4 7 2 9
10 1
0 1 2 3 3 4 5 4 3
8 7 5 9 6 3 4 1 2
10 1
0 1 2 2 4 2 0 4 7
8 9 3 7 5 6 4 2 1
10 1
0 1 2 3 4 5 2 4 0
8 9 7 5 6 4 3 2 1
10 1
0 0 1 1 1 0 5 0 4
2 6 3 7 5 9 8 1 4
10 1
0 1 2 1 0 0 2 6 6
3 7 4 8 9 2 5 6 1
10 1
0 0 1 1 0 2 6 5 0
8 7 3 6 9 2 4 1 5
10 1
0 1 1 0 1 0 6 4 6
5 2 8 4 7 6 9 3 1
10 1
0 0 1 0 0 3 0 0 7
8 2 6 3 4 9 5 1 7
10 1
0 0 0 3 4 5 0 2 6
1 7 8 9 6 2 4 5 3
10 1
0 0 1 2 3 5 3 5 8
9 7 4 2 6 8 3 5 1
10 1
0 1 0 2 0 1 6 7 6
5 4 8 9 3 7 2 6 1
10 1
0 1 2 3 4 0 6 5 5
7 9 6 8 5 3 4 2 1
