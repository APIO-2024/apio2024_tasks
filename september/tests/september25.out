7ckgnn4wyi495puj3ibqf81dqvapyv6b
7
9
9
6
8
9
9
8
9
8
8
9
9
9
9
7
9
8
9
9
36
38
37
39
37
93
998
97
38
38
39
39
39
8
9
9
7
8
9
6
5
7
9
8
8
9
9
8
9
8
8
9
8
